### PR TITLE
instructions for the GitHub version

### PR DESCRIPTION
**Pull request by ghost**
just a simple how to make it available for anyone even here on GitHub.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
 
 Drop it allows one click inserts/uploads of photos among the thousands of photos available on <a href="https://unsplash.com">unsplash.com</a> without leaving the Gutenberg editor.
 
+## How to
+
+After installing please run the following commands to make it work : 
+
+`npm install && npm run build`
+
+Please follow this tutorial https://youknowriad.github.io/gcf/#/contribute if you want to contribute.
```
Comment (youknowriad) [inline README.md]: I think we should remove this line. The sentence above should be enough for now. If the plugin becomes bigger, we'll have to add docs in a similarily to GCF.

File: README.md [hunk 2]
```diff
@@ -2,6 +2,16 @@
 
 Drop it allows one click inserts/uploads of photos among the thousands of photos available on <a href="https://unsplash.com">unsplash.com</a> without leaving the Gutenberg editor.
 
+## How to
+
+After installing please run the following commands to make it work :
```
Comment (youknowriad) [inline README.md]: Can we reword this: 

> Clone this repository to the `wp-content/plugins` folder of your WordPress installation and run the following commands to make it work:

File: README.md [hunk 3]
```diff
@@ -2,6 +2,16 @@
...
 
+## How to
+
+After installing please run the following commands to make it work : 
+
+`npm install && npm run build`
+
+Please follow this tutorial https://youknowriad.github.io/gcf/#/contribute if you want to contribute.
+
+This plugin is available on wordpress.org if you just want to enjoy it without having any command line to run : https://wordpress.org/plugins/dropit/
```
Comment (youknowriad) [inline README.md]: Can we move this to a specific section above the "How To"

`## Install`

File: README.md [hunk 4]
```diff
@@ -2,6 +2,16 @@
 
 Drop it allows one click inserts/uploads of photos among the thousands of photos available on <a href="https://unsplash.com">unsplash.com</a> without leaving the Gutenberg editor.
 
+## How to
```
Comment (youknowriad) [inline README.md]: Can we rename `## Contribute`

File: README.md [hunk 5]
```diff
@@ -2,6 +2,16 @@
...
+
+After installing please run the following commands to make it work : 
+
+`npm install && npm run build`
+
+Please follow this tutorial https://youknowriad.github.io/gcf/#/contribute if you want to contribute.
+
+This plugin is available on wordpress.org if you just want to enjoy it without having any command line to run : https://wordpress.org/plugins/dropit/
+
 ### Features
```
Comment (youknowriad) [inline README.md]: I guess this should also have the same level of heading `## Features`

**Review by youknowriad — COMMENTED** [review]
Thanks for the PR. I left some minor comments but this is great.

File: README.md (hunk 5 above)
Comment (ghost) [reply README.md]: wrong hierarchy then ^^ h1 => h3, h3, no problem to change all the thing to me, just want to add instructions, I'm committing all the thing.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: makes sense

File: README.md (hunk 3 above)
Comment (TweetPressFr) [reply README.md]: done ;)

File: README.md (hunk 4 above)
Comment (ghost) [reply README.md]: done ;)

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: done ;)

File: README.md (hunk 3 above)
Comment (ghost) [reply README.md]: forgot that part ahah sorry. It should be good now.

**Review by youknowriad — APPROVED** [review]
Awesome thank you

**Pull request merged**